MapboxNavigation/NavigationMapView.swift

```diff
@@ -337,10 +337,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         if tracksUserCourse {
             let newCamera = camera ?? MGLMapCamera(lookingAtCenter: location.coordinate, altitude: altitude, pitch: 45, heading: location.course)
             let function: CAMediaTimingFunction? = animated ? CAMediaTimingFunction(name: .linear) : nil
-            let point = userAnchorPoint
-            let padding = UIEdgeInsets(top: point.y, left: point.x, bottom: bounds.height - point.y, right: bounds.width - point.x)
-            // Omit padding when https://github.com/mapbox/mapbox-gl-native/pull/14081 has landed
```

Comment:
This comment was a bit misleading: the actual workarounds were the paddings added in `showcase(_:padding:animated:)` and `fit(to:facing:padding:animated:)`, not the one here. The edge padding here is necessary in order to account for the puck’s uncentered location even within the content frame.

/ref https://github.com/mapbox/mapbox-navigation-ios/issues/2145#issuecomment-503800503